components/Panel/VotePanel/Discussion.tsx

```diff
@@ -1,6 +1,7 @@
 import { Button, LoadingSpinner } from "components";
 import { discordLink, mobileAndUnder, red500 } from "constant";
 import { format } from "date-fns";
+import { enCA } from "date-fns/locale";
```

Comment:
what is this, cant find it in date docs?

Reply:
https://date-fns.org/v2.29.3/docs/Locale
https://en.wikipedia.org/wiki/Date_format_by_country